lstchain/scripts/lstchain_create_run_summary.py

```diff
@@ -0,0 +1,216 @@
+"""
+Create a run summary for a given date containing the number of subruns,
+the start time of the run, type pf the run: DATA, DRS4, CALI, and
+the reference timestamp and counter of the run.
+"""
+
+import argparse
+import logging
+from pathlib import Path
+
+import numpy as np
+import protozfits
+from ctapipe.containers import EventType
+from ctapipe.io import EventSource
+from ctapipe_io_lst.event_time import CENTRAL_MODULE
+from lstchain.paths import parse_r0_filename
+
+# FIXME: take it from somewhere else
+tel_id = 1
+
+
+log = logging.getLogger(__name__)
+
+parser = argparse.ArgumentParser(description="Create run summary file")
+
+parser.add_argument(
+    "-d",
+    "--date",
+    type=str,
+    help="Date for the creation of the run summary in format YYYYMMDD",
+    required=True,
+)
+# TODO: default today()
+
+parser.add_argument(
+    "--r0-path",
+    type=Path,
+    dest="R0_PATH",
+    help="Path to the R0 files. Default is /fefs/aswg/data/real/R0",
+    default=Path("/fefs/aswg/data/real/R0"),
+)
+
+parser.add_argument(
+    "-o",
+    "--output-dir",
+    type=Path,
+    dest="output_dir",
+    help="Directory in which Run Summary file is written",
+    default="./",
+)
+
+
+args = parser.parse_args()
+
+
+def get_list_of_files(date):
+    """Get the list of R0 files from a given date."""
+    path_r0 = args.R0_PATH / date
+    # FIXME: use regular expressions from lstchain.paths.R0_RE
+    list_of_files = path_r0.glob("LST*.fits.fz")
+    return list_of_files
+
+
+def get_list_of_runs(list_of_files):
+    """Get the sorted list of run objects from R0 filenames."""
+    return sorted(parse_r0_filename(file) for file in list_of_files)
+
+
+def get_run_numbers(list_of_runs):
+    """Get the list of run numbers of a given date."""
+    return np.unique([run_info.run for run_info in list_of_runs])
```

Comment:
`np.unique` can also directly return you the counts for each unique item. That would give you the number of subruns, right?

Reply:
Yes, I've done that in the same function.